scripts/validate-front-matter.js

```diff
@@ -102,7 +102,7 @@ readdir("_source/_posts", (err, files) => {
     validateBy(files[i], content.attributes.by);
     validateComms(files[i], content.attributes.communities);
     validateTypes(files[i], content.attributes.type);
-    validateTags(files[i], content.attributes.tags);
+    //validateTags(files[i], content.attributes.tags);
```

Comment:
@mraible not sure if this can be merged with this change. Adding the comment for the sake of visibility!

Reply:
Good catch! I re-enabled it on my PR to your PR in [this commit](https://github.com/dbikic/okta-blog/pull/1/commits/af2e1150197771a3b27e6a0b5b62c3026f210da6).